src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixtures/mainIdeWindow/toolWindowsPane/ToolWindowsPane.java

```diff
@@ -42,211 +36,119 @@
 @FixtureName(name = "Tool Windows Pane")
 public class ToolWindowsPane extends CommonContainerFixture {
     private RemoteRobot remoteRobot;
-    private static String lastBuildStatusTreeText;
 
     public ToolWindowsPane(@NotNull RemoteRobot remoteRobot, @NotNull RemoteComponent remoteComponent) {
         super(remoteRobot, remoteComponent);
         this.remoteRobot = remoteRobot;
     }
 
     /**
-     * Create fixture for the Stripe button
-     *
-     * @param label label text of the stripe button
-     * @return fixture for the Stripe button
+     * Open project explorer
      */
-    public JButtonFixture stripeButton(String label) {
-        return button(byXpath("//div[@text='" + label + "']"), Duration.ofSeconds(2));
+    public void openProjectExplorer() {
```

Comment:
there is an opportunity to return the appropriate fixture for every open* method

Reply:
Fixed.